packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -772,6 +773,17 @@ export default class TabAdvanced extends Component {
                     </td>
                   </tr>
                 )}
+                {vm.boot.firmware === 'uefi' && (
+                  <tr>
+                    <th>{_('secureBoot')}</th>
+                    <td>
+                      <Toggle
+                        value={vm.secureBoot === 'true' ? true : false}
```

Comment:
Shouldn't the `=== 'true'` be in `xapi-object-to-xo.js` instead? It looks like that's what we usually do.